README.md

```diff
@@ -18,99 +18,56 @@ These instructions will get you a copy of the project up and running on your loc
-3. Ensure that you add %JAVA_HOME%\bin to your system PATH
-4. Install Eclipse, such as the "Eclipse IDE for Enterprise Java Developers"
-5. Install gcc, gdb, and msys (e.g. download and run mingw-get-setup.exe from [here](https://osdn.net/projects/mingw/releases/) and then use the package installer to install C and C++, GDB, MSYS, and any other packages you may want.)
-6. Ensure your gcc is accessible from the command line (e.g. add C:\MinGW\bin to the system PATH)
-7. In Eclipse, and install the C Development Tools (CDT)
-    1. In Eclipse, choose the menu option Help > Install New Software
-    2. Choose to work with the main repository (e.g. 2020 - 06 - http://download.eclipse.org/releases)
-    3. Under Programming Languages, choose C/C++ Autotools, C/C++ Development Tools, C/C++ Development Tools SDK, C/C++ Library API Documentation Hover Help, and C/C++ Unit Testing Support
-    
-### Working with the Code in the Development Environment
-
-1. Copy the HTTPS clone URL into the copy/paste buffer
-     1. In this repository, the "Code" button provides the [HTTPS clone](https://github.com/graph-algorithms/edge-addition-planarity-suite.git) link to use to get the code.
-2. Start by making a new eclipse workspace for 'graph-algorithms'
-     1. You may need to select 'Prompt for Workspace on startup' in Window | Preferences | General | Startup and Shutdown | Workspaces, then close and reopen eclipse
-     2. In the initial workspace dialogue, one can specify a new folder that gets created, e.g. c:\Users\\_you_\Documents\eclipse\workspaces-cpp\graph-algorithms
-3. Click 'Checkout projects from Git' in the new workspace Welcome page 
-4. Pick 'Clone URI' and hit Next
-5. The URI, Host, and Repository are pre-filled correctly from the copy/paste clipboard.
-6. Leave the User/Password blank (unless you have read/write access to the project), and hit Next
-7. The master branch is selected by default, so just hit Next again
-8. Change the destination directory to a subdirectory where you want to store the project code (e.g. c:\Users\\_you_\Documents\eclipse\workspaces-cpp\graph-algorithms\edge-addition-planarity-suite)
-9. Hit Next (which downloads the project), Hit Next again (to Import existing Eclipse projects), Hit Finish
-
-Now that the project is available, the code can be built and executed:
-
-1. Open the C/C++ Perspective
-     1. Use the Open Perspectives button (or use Windows | Perspective | Open Perspective | Other...)
-     2. Select C/C++
-2. Right-click Planarity-C project, Build Configurations, Build All
-3. Right-click Planarity-C project, Build Configurations, Set Active, Release
-4. Right-click Planarity-C project, Run As, Local Application, planarity.exe (release)
+Please refer to the [2. Dev Setup](https://github.com/graph-algorithms/edge-addition-planarity-suite/wiki/2.-Dev-Setup) wiki page for instructions on how to install development dependencies on Windows 10 22H2, Debian 12.70, or MacOS Sequoia 15.0, as
```

Comment:
The readme should probably leave the versions of OS's to the wiki page. Also, on the wiki page, I think it's going to be reasonable to include Windows 11 not just Windows 10 22H2 (but that is a discussion about the wiki page that can happen outside of PR land).